sfcasts/dashboard.md

```diff
@@ -1,95 +1,61 @@
+
+The `DashboardController` is just a normal controller, but it extends `AbstractDashboardController`. I'll hold "Cmd" and open that. 
+
+This implements `DashboardControllerInterface`. So this is a controller, but by implementing the interface, it gives it special superpowers. There are a number of methods inside of here that we can override to configure what our dashboard looks like. We'll be doing that little by little throughout this tutorial. 
+
+Because this is just a normal route in controller, there's really nothing special about it. It's not automatically secured and there's no security being applied to this. I mean, check it out: I'm not even logged in right now, and I am successfully on my admin dashboard.
+
+So let's secure it! I'll also do this with annotations. I already have SensioFrameworkExtraBundle installed. So I can say `#[IsGranted()]` and hit "tab" to auto complete that. Now we'll look for `ROLE_ADMIN`. That's kind of my base admin role, so all admin users will have that role. Now when I refresh... Beautiful! We bounced back over to the login page!
+
+So to log in, if you open `src/DataFixtures/AppFixtures`, I have a bunch of dummy users in the database. There is a super admin, a normal admin, and then somebody known as a moderator. We'll talk more about those later, when we get deeper into how to secure different parts of your admin with different roles.
+
+I'll log in with "admin@example.com"... password "admin password"...
+
+And... beautiful! Now we're back to our dashboard!
+
+Of course, if you want to, instead of securing `IsGranted` with the PHP attribute, you can use `$this->denyAccessUnlessGranted()`. You can also go to `config/packages/security.yaml` and, down at the bottom, you can always just use access control for this - whichever one you like better.
+
+So, our dashboard is the jumping off point for our admin, but there's nothing too special here. This page has a title, some menu items, and a nice little user menu over here. Eventually, we'll be able to actually generate something cool on this page, like links or graphs or something. By the way, styling is all done in Bootstrap 5 with FontAwesome.
+
+So let's see if we can customize the dashboard a little bit. One of the best things about EasyAdmin is that all the config is done in PHP, usually via methods in your controller. So, for example, want to configure the dashboard? There is a configured dashboard method for that!
+
+We can change the title of the page to call during overflow admin. And I wanna refresh, we get Coran overflow admin, and there are a number of other methods. You can just look at the auto complete here. So, related to the fab icon path, there's various ways that you can modify things. You can see there's things about the sidebar being minimized. That's referring to a nice little feature here where you can double click on this little thing to expand or collapse the sidebar.
```

Comment:
Danielle: This one really lost me and I haven't made changes to every sentence because I didn't know what changes to make. Does "overflow admin" need formatting of some kind? What is "Coran"? I think the last two sentences are mostly ok, though

Reply:
This is a spot where you'd need the video :). The name of our site is `Cauldron Overflow` - that's the "call during overflow" you're hearing ;) 

```diff
- And I wanna refresh, we get Coran overflow admin, and there are a number of other methods
+ And when I refresh, we see Cauldron Overflow Admin. And there are a number of other methods.
```

Apart from "I wanna" being "when I", the most important thing here is that there are 2 separate ideas. In the first sentence, I'm refreshing the page. Then, there's a pause while I go back to the editor and start talking about the next topic. That's why I split it into 2 sentences

Reply:
"related to the fab icon path" -> "related to fav icon path". 

In the video, you can see my editor showing me available methods. One is called `setFavIconPath`. So that's the hint for the correct text here. I didn't use the technical word `setFavIconPath` - since I say "related to fav icon path"... I'm just saying that there is a method "about this topic", not "here is the exact method name".

Reply:
I wondered about that, because FontAwesome uses the term "fab" and I was trying to figure out, without the video context, whether it was "fab" or "fav". Thanks!